tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraLongLivedTokenSetMethodContract.scala

```diff
@@ -633,4 +633,322 @@ trait LinagoraLongLivedTokenSetMethodContract {
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response).isEqualTo(
+      s"""{
+         |  "sessionState":"${SESSION_STATE.value}",
+         |  "methodResponses": [
+         |    ["error", {
+         |      "type": "unknownMethod",
+         |      "description": "Missing capability(ies): urn:ietf:params:jmap:core, com:linagora:params:long:lived:token"
+         |    },"c1"]
+         |  ]
+         |}""".stripMargin)
+  }
+
+  @Test
+  def setDestroyShouldFailWhenWrongLongLivedTokenId(): Unit = {
+    val request: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "destroy": ["wrong@id"]
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val response: String = `given`
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response)
+      .inPath("$.methodResponses[0][1]")
+      .isEqualTo(
+        """
+           |{
+           |  "type": "invalidArguments",
+           |  "description": "{\"errors\":[{\"path\":\"obj.destroy[0]\",\"messages\":[\"LongLivedTokenId does not match Id constraints: Predicate failed: 'wrong@id' contains some invalid characters. Should be [#a-zA-Z0-9-_] and no longer than 255 chars.\"]}]}"
+           |}""".stripMargin)
+  }
+
+  @Test
+  def setDestroyShouldSucceedWhenUnknownLongLivedTokenId(): Unit = {
+    val unknownTokenId: String = LongLivedTokenId.generate.value.toString
+
+    val request: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "destroy": ["$unknownTokenId"]
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val response: String = `given`
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response).isEqualTo(
+      s"""{
+         |    "sessionState": "${SESSION_STATE.value}",
+         |    "methodResponses": [
+         |        [
+         |            "LongLivedToken/set",
+         |            {
+         |                "accountId": "$ACCOUNT_ID",
+         |                "destroyed": ["$unknownTokenId"]
+         |            },
+         |            "c1"
+         |        ]
+         |    ]
+         |}""".stripMargin)
+  }
+
+  // TODO: calling /token endpoint after long lived token revocation should fail (waiting for /token endpoint)
```

Comment:
The build seems not stable yet in https://github.com/linagora/tmail-backend/pull/234 so I preferred not rebasing on it for now